src/c_buttons.hpp

```diff
@@ -21,7 +22,7 @@
   int lastLoopKey = -1;
 
   #if LCD_BUTTON_TEST == 1
```

Comment:
We should add the button test to the matrix build to ensure this won't break in the future

Reply:
Yeah, agreed. @andre-stefanov would this be easy to do?